pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,102 @@
+---
+$title: Use AMP components in non-AMP pages
+$order: 1
+description: 'Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP.'
+tutorial: false
+toc: true
+formats:
+  - websites
+author: CrystalOnScript
+---
+
+Use a subset of AMP components as standalone elements in non-AMP pages with Bento AMP. Bento AMP is a project that embraces use of AMP components without the need to commit to fully valid AMP. This creates copy and past-able code for common UI widgets, allows testing if AMP is the right switch for your website, and using Bento in a migration plan. 
+
+[tip type="important"]
+ Bento is only available [experimentally](../learn/experimental.md). Read our [Blog Post](TODO) for information on future plans and how to get involved! 
+[/tip]
+
+# Use Bento components in non-AMP pages
+
+Bento components are available for experimental use in the developer preview.
```

Comment:
Can you elaborate on what this means? Is it OK to use these in production? What are the risks? 

//cc @dvoytenko @caroqliu

Reply:
They are guarded behind an experimental flag which can be enabled on the document or in the console, and this is no different from other [experiments in AMP](https://amp.dev/documentation/guides-and-tutorials/learn/experimental/).

The risks are also the same - it is experimental because we reserve the right for the API to change drastically if given feedback. This is not likely IMO but the purpose of the preview is to get feedback that can include making breaking changes if it makes the features better.

Reply:
updated the wording, lmkwyt!

Reply:
Thanks Caroline. Might be worth mentioning in the guide.